web/config/sync/core.extension.yml

```diff
@@ -6,6 +6,7 @@ module:
   dynamic_page_cache: 0
   field: 0
   filter: 0
+  help: 0
```

Comment:
[Adminimal depends on the help module](https://www.drupal.org/node/2629744) so enabling the help module is a requirement. I have updated the commit comment to explain this.